packages/nimble-components/src/wafer-map/tests/wafer-map-matrix.stories.ts

```diff
@@ -179,6 +180,12 @@ const componentWaferWithGridDimensions = ([
 >
 </${waferMapTag}>`;
 
+const componentWaferMapWorkerCanvas = (): ViewTemplate => html`<${waferMapTag}
+    :diesTable="${() => wafermapDiesTableSets[0]}"
+    :apiVersion="${() => 'experimental'}"

```

Comment:
The wafer-map class does not have a property named apiVersion. I don't think this does anything and this line can be removed

Reply:
You are right! I deleted it